src/main/java/com/epam/training/auction_backend/services/AuctionsServiceImpl.java

```diff
@@ -12,28 +14,38 @@
 import com.epam.training.auction_backend.entity.User;
 
 public final class AuctionsServiceImpl implements AuctionsService {
-	
-	private static final Logger logger = Logger.getLogger(AuctionsServiceImpl.class);
-	
-	@Autowired
-	private AuctionRepository repository;
-	
-	public List<AuctionTransferObject> getActiveAuctions() {
-		return repository.findByIsActive(true);
-	}
-
-	public List<AuctionTransferObject> getArchivedAuctions() {
-		return repository.findByIsActive(false);
-	}
-
-	public void addAuction(AuctionTransferObject auctionTransferObject) {
-		User seller = new User(auctionTransferObject.getSeller().getUsername(), auctionTransferObject.getSeller().getPassword());
-		Auction auction = new Auction(auctionTransferObject.getTitle(), auctionTransferObject.getDescription(), seller);
-		repository.save(auction);
-		logger.info("Book " + auction.getTitle() + " is successfully added to the auction.");
-	}
-
-	public Optional<AuctionTransferObject> getAuctionById(long id) {
-		return repository.findOne(id);
-	}
+
+    private static final Logger logger = Logger.getLogger(AuctionsServiceImpl.class);
+
+    @Autowired
+    private AuctionRepository repository;
```

Comment:
It's better to use DI via constructor in this case because repository here is mandatory